modules/tinymce/CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The autocompleter now supports a multiple character trigger using the new `trigger` configuration #TINY-8887
 - The formatter will now apply some inline formats like color and font size to list item elements when the entire item content is selected. #TINY-8961
 - The plugin lists in the Help dialog are now sorted alphabetically. #TINY-9019
+- Alignment can now be applied to more types of embedded media elements. #TINY-8687
```

Comment:
Not sure about this changelog item. I don't want to add details about enhanced media embed and not make it specific to the media plugin since it's really not. Or should it be split up into one for media plugin elements and one for ephox embedded elements. Suggestions are welcome.

Reply:
This seems fine!